Actors/ActorResolver.cs

```diff
@@ -1,191 +1,192 @@
-using Dalamud.Plugin.Services;

```

Comment:
Meh, I have no idea why this whole file is marked as changed but it's not for me locally.

Reply:
It's those stupid line endings, lemme see if I I can fix that

Reply:
I have done all of this already anyway.

Reply:
Oh, nice! Was a good learning exp anyway

Reply:
Sorry if this came off as rude. I still appreciate you trying to help out!

Reply:
lol, don't worry it didn't come across that way at all :D i appreciate the comment though